### DISCUSSION TITLE
Clarification on the `sideload` possible payloads

### DISCUSSION THREAD

**Discussion by 3m0W33D** [BishopFox/sliver#1371] [Q&A] [answered]
Can I clarify what are the accepted executables that are supposed to work with `sideload` call?
For windows is DLLs the only accepted payload?
For linux is only a shared objected accepted? Is it possible for linux to use compiled executable to be sideloaded?

**Comment by rkervella** [comment, marked as answer]
For Windows, either a DLL or plain executable (not documented but supported via Donut) PE will work. For Linux and MacOS, only shared libraries will work. Relevant wiki page: https://github.com/BishopFox/sliver/wiki/Using-3rd-party-tools#shared-libraries-side-loading

**Reply by 3m0W33D** [reply]
Hi in the documentation shown above, all arguments are specified within -a option but in the version of sliver that I am using arguments are just appended to the back of the command.
![image](https://github.com/BishopFox/sliver/assets/26686523/e6342c1c-9cf2-43e8-bb07-5f448c5640a2)

Can I confirm which is the correct syntax to perform a sideload?
